src/mpyl/reporting/formatting/markdown.py

```diff
@@ -16,14 +18,24 @@ def summary_to_markdown(summary: TestRunSummary):
            f"💔 {summary.errors} 🙈 {summary.skipped}"
 
 
+def __add_link(name: str, output: Output) -> str:
+    if output.produced_artifact and output.produced_artifact.artifact_type == ArtifactType.DEPLOYED_HELM_APP:
+        url = output.produced_artifact.spec.get(DEPLOYED_SERVICE_KEY)
+        if url:
+            return f'[{name}]({url}/swagger/index.html)'
+
+    return name
+
+
 def __to_oneliner(result: list[StepResult], plan: set[Project]) -> str:
     project_names: list[str] = []
     if plan:
         sorted_plan = sorted(plan, key=operator.attrgetter('name'))
         for proj in sorted_plan:
             found_result = next((r for r in result if r.project.name == proj.name), None)
             if found_result:
-                project_names.append(f'*{proj.name}*' if found_result.output.success else f'~~{proj.name}~~')
+                project_names.append(f'*{__add_link(proj.name, found_result.output)}*' if found_result.output.success
```

Comment:
this seems like it could be optimised by just making the above function handle all the `if else` logic around name vs link

Reply:
I hope this makes it a bit simpler fae82c3032e9dbe3ac770c58153e32f0adbddf1a